v3.0/tidb-in-kubernetes/initialize-cluster.md

```diff
@@ -34,9 +34,19 @@ aliases: ['/docs-cn/v3.0/how-to/configure/initialize-cluster/']
     kubectl create secret generic tidb-secret --from-literal=root=<root-password> --from-literal=developer=<developer-passowrd> --namespace=<namespace>
     ```
 
-    该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。
+    该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。并且创建的普通用户developer默认只有USAGE权限，其他权限请在tidb.initSql中设置。
```

Comment:
- Keep the format consistent.
- Leave a space between Chinese and English characters.

```suggestion
    该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。并且创建的普通用户 `developer` 默认只有 `USAGE` 权限，其他权限请在 `tidb.initSql` 中设置。
```